tests/fixtures/configuration.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var Configuration = require('../../lib/configuration.js');
+
+var FakeConfiguration = function(config) {
+  return Configuration.bind(this)(config, { warn: function () {} });
```

Comment:
Prefer: `Configuration.call(this, config, { warn: function() {} })` to immediately invoking a bound function.